Source/WebCore/rendering/svg/RenderSVGImage.cpp

```diff
@@ -132,8 +132,7 @@ void RenderSVGImage::paint(PaintInfo& paintInfo, const LayoutPoint& paintOffset)
         return;
 
     if (paintInfo.phase == PaintPhase::ClippingMask) {
-        // FIXME: [LBSE] Upstream SVGRenderSupport changes
-        // SVGRenderSupport::paintSVGClippingMask(*this, paintInfo);
+        SVGRenderSupport::paintSVGClippingMask(*this, paintInfo);
```

Comment:
I know I put this into SVGRenderSupport in LBSE downstream, but I'd vote to move this into a shared class, maybe even RenderLayerModelObject?

Reply:
Not sure, will it be used from non SVG? Added a FIXME for now.

Reply:
No, but RenderLayerModelOBject is the only common renderer that is the base of all SVG renderers.